Publish GridFTP and GSI transition plan (#705)
opensciencegrid/technology: docs/policy/gridftp-gsi-migration.md

| Aug 2019  | Including HTCondor 8.9.2 in the ‘upcoming’ repository (first HTCondor version with SciTokens support). | &#9989;       |
| Oct 2019  | OSG no longer carries OSG-specific patches for the GCT.  All patches are upstreamed or retired.        |               |
| Jan 2020  | "GSI free" site demo. Show, at proof-of-concept / prototype level, all components without use of GCT.  |               |
| July 2020 | All GCT-free components are in OSG-Upcoming.                                                           |               |
| Jan 2021  | OSG series 3.6, without GCT dependencies, is released.                                                 |               |
| Jan 2022  | End of support for OSG 3.5.                                                                            |               |

Frequently Asked Questions
--------------------------

### How does SciTokens interoperate with other token technologies in the WLCG? ###

We have been very active with the WLCG AuthZ working group and the WLCG JWT profile incorporates many core pieces of

> SciTokens -- enough that we feel confident in saying that we will support the relevant parts of WLCG JWT in the

Reviewer: A bit informal.  Given 0.4.0 came out in the meantime, perhaps:

> The `scitokens-cpp` library used by OSG can transparently use both WLCG JWTs and SciTokens.

SciTokens client library for existing adopters.

### Will a US-LHC migration from GridFTP to XRootD require the same migration for WLCG? ###

Europe has been an enthusiastic partner in doing this switch.

> That said, there are a number of options to disentangle the two if the pace of evolution turns out to be very different.

Reviewer: We have been working to ensure to coordinate our activities through the WLCG DOMA group.

Author: It's phrased as a yes/no question so I've changed this answer to:

> No but we have been working to ensure that we coordinate our activities through the WLCG DOMA group.

### What role does LCMAPS play with SciTokens? ###

LCMAPS only works with GSI.

> The model for SciTokens is sufficiently more simple for sites that the full complexity of LCMAPS is a bit overkill.

Reviewer: ```suggestion
The model for SciTokens is sufficiently more simple for sites that the full complexity of LCMAPS is not needed.
```

### What does a SciTokens transition for GlideinWMS mean for European sites? ###

The only piece that involves European sites is the factory to CE relationship:

> given HTCondor-CE is covered, we need to focus on ARC-CE.

Reviewer: ```suggestion
given HTCondor-CE 4.0 already supports SciTokens, we have begun to engage with the ARC-CE team.
```
